feincms_extensions/content_types.py

```diff
@@ -0,0 +1,50 @@
+from feincms.content.medialibrary.models import MediaFileContent
+from feincms.content.richtext.models import RichTextContent
+from feincms.content.section.models import SectionContent
+
+
+class JsonRichTextContent(RichTextContent):
+    class Meta(RichTextContent.Meta):
+        abstract = True
+
+    def json(self, **kwargs):
+        """Return a json serializable dictionary containing the content."""
+        return {'html': self.text}
+
+
+class JsonSectionContent(SectionContent):
+    class Meta(SectionContent.Meta):
+        abstract = True
+
+    def json(self, **kwargs):
+        """Return a json serializable dictionary containing the content."""
+        mediafile = self.mediafile
+        return {
+            'title': self.title,
+            'html': self.richtext,
+            'mediafile': {
+                'url': mediafile.file.url,
+                'type': mediafile.type,
+                'created': mediafile.created,
+                'copyright': mediafile.copyright,
+                'file_size': mediafile.file_size,
+            },
```

Comment:
Looks like this could be done with a serializer, however the worry here is the nesting while we're still on drf2. What are your thoughts @Ian-Foote @meshy